google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/RpcViews.java

```diff
@@ -56,19 +40,25 @@ public static void registerBigtableClientGfeViews() {
     registerBigtableClientGfeViews(Stats.getViewManager());
   }
 
+  public static void registerAllBigtableClientViewsWithExtraTags() {
+    registerViewsWithExtraTags(Stats.getViewManager());
+  }
+
```

Comment:
This definitely needs documentation and probably a better method name :)

Reply:
I think it might be better to treat this as metric schema versions. So maybe an enum for the versions and each vartiant can describe what it incldues?